src/components/NcSelect/NcSelect.vue

```diff
@@ -1263,12 +1265,14 @@ body {
 
 	.vs__selected-options {
 		// If search is hidden, ensure that the height of the search is the same
-		min-height: 40px; // 36px search height + 4px search margin
+		min-height: auto; // 36px search height + 4px search margin
```

Comment:
So this min-height is unrelated?

Reply:
Yeah, that's the min-height of the dropdown with the options